doctr/models/builder.py

```diff
@@ -288,6 +290,10 @@ def __call__(
         if len(boxes) != len(text_preds) or len(boxes) != len(page_shapes):
             raise ValueError("All arguments are expected to be lists of the same size")
 
+        if not orientations:
```

Comment:
better to be specific:
```python
if not isinstance(orientations, list):
```

Reply:
took the suggestions from the comment after this one :+1: